### PR TITLE
Barebones microsite

### PR DESCRIPTION
**Pull request by denisrosset**
Followup on #753 
Reviewed by @larsrh

### CODE REVIEW

**Comment by denisrosset** [comment]
Yep, it was intentional to move the whole README.md to the site, as it removes some `tut` machinery that was checking it before.
Agree we need a barebones README.md though. Will do and merge.

**Comment by denisrosset** [comment]
Done. Waiting for @non input, as I can't publish a microsite on non/spire; right now, it's pointing to https://denisrosset.github.io/spire/ which is not ideal.
I'd love to move Spire to the typelevel umbrella -- @larsrh how would we deal with the microsite in that case?

**Comment by larsrh** [comment]
Hmm, both of us should be able to just create a gh-pages branch in this repository, which is all that is required to create a microsite.

**Comment by denisrosset** [comment]
Let me try.

**Comment by denisrosset** [comment]
Fighting with SBT's stupid cache..

**Comment by denisrosset** [comment]
@non @larsrh The gh-pages branch is there, but not accessible ( https://non.github.io/spire/ returns a 404 ). I guess we have to activate the site in the project settings.

File: build.sbt [hunk 1]
```diff
@@ -309,6 +309,85 @@ lazy val commonJvmSettings = Seq(
...
 
+lazy val docsMappingsAPIDir = settingKey[String]("Name of subdirectory in site target directory for api docs")
+
+lazy val docSettings = Seq(
+  scalacOptions in Tut := (scalacOptions in Tut).value.filterNot(Set("-Ywarn-unused-imports", "-Xlint").contains),
+  micrositeName := "Spire",
+  micrositeDescription := "Powerful new number types and numeric abstractions for Scala",
+  micrositeAuthor := "Spire contributors",
+  micrositeHighlightTheme := "atom-one-light",
+  micrositeHomepage := "http://non.github.io/spire",
```
Comment (larsrh) [inline build.sbt]: This should be `typelevel` now :tada:

**Comment by larsrh** [comment]
This is live now: https://typelevel.org/spire/

**Pull request merged**